REVIEW_GUIDELINES.md

```diff
@@ -54,7 +88,19 @@ selector:
   release: {{ .Release.Name }}
 ```
 
-If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`). The `component` label must be added to label selectors as well.
+If a chart has multiple components, a `component` label should be added to the selector (see above).
+
+### Persistence labels
```

Comment:
Should we add PVC? If so, does anybody have suggestion?

Reply:
if add the matchlabels to the PVC will break and the PVC will never get the PV. only if you set the pv with the lables as well. but I guess we are not generating the PV manifest.

Reply:
So should I simply state something like "Unless special needs, you should not set PVC matchLabels because it would prevent automatic PV provisionning"?

Reply:
Done. Is that okay for you?

Reply:
looks great to me :) thanks!!